doctr/datasets/mjsynth.py

```diff
@@ -60,10 +60,10 @@ def __init__(
         set_slice = slice(train_samples) if self.train else slice(train_samples, None)
 
         for path in tqdm(iterable=img_paths[set_slice], desc='Unpacking MJSynth', total=len(img_paths[set_slice])):
-            label = [path.split('_')[1]]
+            label = path.split('_')[1]
```

Comment:
so the label used to be a list of string? (with a single element)

Reply:
Before: ['xyz'] 
After: 'xyz' :)

Reply:
Yes, but I don't get how "before" used to work :thinking: 
Why would we have a list of string? Is it because we weren't thorough enough in the unittest and it's a mistake? Or I'm missing something?

Reply:
Mmmhh no unitests are fine but we have had no case where a dataset for recognition was used (I have introduced this a bit earlier).. first is the eval_recognition script with:
targets = [t['labels'][0] for t in targets]

After updating the __getitem__ this will be the same without the list indexing :) in this case i can revert the test changes in this PR

At the end we will have the following signature: Tuple[Tensor, Dict[str, Union[str, np.ndarray]] does this match with your idea ? Like (image, {'labels': 'str, 'boxes': array} (boxes only if available)